inverse/beamformer_dics.m

```diff
@@ -81,28 +82,32 @@
 powmethod      = ft_getopt(varargin, 'powmethod');  % the default for this is set below
 realfilter     = ft_getopt(varargin, 'realfilter'); % the default for this is set below
 subspace       = ft_getopt(varargin, 'subspace');
+
 % these settings pertain to the forward model, the defaults are set in compute_leadfield
```

Comment:
what about backproject, weight?

Reply:
Yes, well there indeed I might have missed it while fixing the conflicts, but I didn't find it back in the code from master that I merged into the PR

Reply:
you are right, these two are in the help but not in the actual code. My bad!

Reply:
OK, I have now added weight and backproject as options for ft_compute_leadfield, using the same heuristic as in ft_prepare_leadfield, for passing on the options to the lower level function. While I was at it, I also added a similar heuristic for the options that go into ft_inv